src/TwilioVideo.android.js

```diff
@@ -277,17 +279,18 @@ class CustomTwilioVideoView extends Component {
     this.runCommand(nativeEvents.toggleSoundSetup, [speaker]);
   }
 
-  captureFrame() {
-    this.runCommand(nativeEvents.captureFrame, []);
+  captureFrame(filename) {
```

Comment:
this is the only actual change in this file. accidentally used save with format so it formatted quite a bit of this file

Reply:
I ran into this in the last PR I made to `react-native-twilio-video-webrtc`.  One good point that @garyp  [brought up](https://github.com/streem/react-native-twilio-video-webrtc/pull/16#discussion_r1372589080) is that while this is good practice for our code, we may not want to introduce too many formatting changes to this lib's code that might make merging into the upstream more annoying in the future.

Reply:
And vice versa too: we'll likely want to pull changes from upstream into our fork to keep up to date on any fixes/improvements they make. Too many formatting changes make that process a lot more annoying because you'll end up with lots of merge conflicts.